arci/src/clients/joint_trajectory_clients_container.rs

```diff
@@ -207,4 +207,101 @@ mod tests {
             "[\"part1\", \"high\", \"part2\", \"low\", \"part3\", \"high\", \"part4\", \"middle\"]"
         );
     }
+
+    #[test]
+    fn test_container_current_pos_ok() {
+        use super::*;
+        #[derive(Debug, Clone)]
+        struct Dummy {
+            name: Vec<String>,
+            pos: Vec<f64>,
+        }
+        impl JointTrajectoryClient for Dummy {
+            fn joint_names(&self) -> &[String] {
```

Comment:
I think you can move this part into `mod test`.
I mean L106 and reuse it everywhere.
Copy/Paste is not nice.